src/rdkafka_fetcher.c

```diff
@@ -402,9 +403,9 @@ rd_kafka_fetch_reply_handle_partition(rd_kafka_broker_t *rkb,
                                            "fetch response: ignoring.",
                                            RD_KAFKAP_STR_PR(topic),
                                            hdr.Partition, AbortedTxnCnt);
-
-                                rd_kafka_buf_skip(rkbuf,
-                                                  AbortedTxnCnt * (8 + 8));
+                                rd_kafka_buf_skip(rkbuf, (8 + 8));
+                                /* AbortedTransaction tags */
+                                rd_kafka_buf_skip_tags(rkbuf);
```

Comment:
there was a for loop here.
Also `k `definition should go to the top of the function.

```suggestion
                                for (k = 0; k < AbortedTxnCnt; k++) {
                                        rd_kafka_buf_skip(rkbuf, (8 + 8));
                                        /* AbortedTransaction tags */
                                        rd_kafka_buf_skip_tags(rkbuf);
                                }
```